fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/scalar/Timestamp.java

```diff
@@ -41,9 +42,15 @@ public class Timestamp extends ScalarFunction
     // This preference follows original planner. refer to ScalarType.getDefaultDateType()
     public static final List<FunctionSignature> SIGNATURES = Config.enable_date_conversion ? ImmutableList.of(
             FunctionSignature.ret(DateTimeV2Type.SYSTEM_DEFAULT).args(DateTimeV2Type.SYSTEM_DEFAULT),
+            FunctionSignature.ret(DateTimeV2Type.SYSTEM_DEFAULT).args(DateTimeV2Type.SYSTEM_DEFAULT,
+                    VarcharType.SYSTEM_DEFAULT),
+            FunctionSignature.ret(DateTimeType.INSTANCE).args(DateTimeType.INSTANCE, VarcharType.SYSTEM_DEFAULT),
             FunctionSignature.ret(DateTimeType.INSTANCE).args(DateTimeType.INSTANCE)
```

Comment:
please groups by arg count, e.g. put 1 arg signatures togather and 2 args signatures togather